src/pfe/portal/modules/Templates.js

```diff
@@ -275,6 +230,39 @@ module.exports = class Templates {
   }
 }
 
+async function getTemplatesFromRepo(repository) {
+  if (!repository.url) {
+    throw new Error(`repo '${repository}' must have a URL`);
+  }
+  const repoUrl = new URL(repository.url);
+
+  if (!repoUrl.host.includes('github')) {
+    throw new Error(`URL '${repository.url}' must be a GitHub repo`);
```

Comment:
Is this really a requirement? is there any particular reason for it?

Reply:
sure, I can change this

Reply:
updated